Resources/public/css/views/dashboard/allcontent-dashboard-block.css

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+.ez-view-dashboardblockallcontentview .pure-table {
+    width: 100%;
+    border: 0;
+    table-layout: fixed;
+}
+
+.ez-view-dashboardblockallcontentview [class*="ez-allcontent-block-head--"] {
+    box-sizing: border-box;
+    padding: .5em;
+}
+
+.ez-view-dashboardblockallcontentview .ez-allcontent-block-wrapper {
+    box-sizing: border-box;
+    border: 1px solid;
+    position: relative;
+}
+
+.ez-view-dashboardblockallcontentview.is-loading .ez-allcontent-block-content {
+    height: 3em;
+    display: block;
+}
+
+.ez-view-dashboardblockallcontentview .ez-allcontent-block-head--title,
+.ez-view-dashboardblockallcontentview .ez-allcontent-block-cell--title { width: 30%; }
```

Comment:
isn't it enough to set the width of the head in order to set the width of the column ?